copy/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "radicle-copy"
+version = "0.1.0"
+authors = ["The Radicle Team <dev@radicle.xyz>"]
+edition = "2018"
+license = "GPL-3.0-or-later"
+
+[dependencies]
+# FIXME: Move to radicle-bins
+anyhow = "1.0"
```

Comment:
We may want to loosen the constraint to "1" -- we depend on `anyhow` elsewhere with this bound, and it's (too) easy to make `cargo` pull in more than one version of the same crate.